pages/who-we-are/index.js

```diff
@@ -0,0 +1,56 @@
+import { Disclosure } from "@headlessui/react";
+import { FaAngleDown } from "react-icons/fa";
+import Heading from "../../components/Heading";
+
+const FAQS = [
+  {
+    prompt: "Do I need technical knowledge to join?",
+    content:
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Leo nec vitae ac faucibus. Blandit ultrices magna natoque enim quis suscipit. Vitae eget venenatis rhoncus aliquet curabitur mauris, sed turpis nulla. Neque molestie mi placerat ultrices sit in sit pretium.",
+  },
+  {
+    prompt: "How much time should I expect to commit per week?",
+    content:
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Leo nec vitae ac faucibus. Blandit ultrices magna natoque enim quis suscipit. Vitae eget venenatis rhoncus aliquet curabitur mauris, sed turpis nulla. Neque molestie mi placerat ultrices sit in sit pretium.",
+  },
+  {
+    prompt: "Question 3",
+    content:
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Leo nec vitae ac faucibus. Blandit ultrices magna natoque enim quis suscipit. Vitae eget venenatis rhoncus aliquet curabitur mauris, sed turpis nulla. Neque molestie mi placerat ultrices sit in sit pretium.",
+  },
+];
```

Comment:
In my PRs I made a `data` folder with all this sort of data. Let's move it all there: https://github.com/Code-the-Change-YYC/code-the-change-yyc-site/pull/27/files#diff-35e112640a80a496564cdf54f8809499e3bfffc261e8b3e5c0c726ad13d02762R1

So in the root we'll have `data/faq.js' or something